test/queries/index.test.js

```diff
@@ -1,3 +1,4 @@
+exports.addUser = require('./addUser.test');
```

Comment:
```suggestion
require('./addUser.test');
```
just commit it directly from here